feature/security/gnsi/authz/feature.textproto

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id {
+  name: "system_authz"
```

Comment:
good catch. I updated the name.

regarding the dir, I see there is a security in the experimental, and gNSI is solely a security thing, therefore I put it under security folder. 

Is this dir thing a blocking comment? i.e we can easily move it later if we don't find it convenient, right?

Reply:
I'll let you decide - it isn't impossible to move later.  I just prefer we stick to a consistent pattern of where we things.

You updated it to gnoi_systemz, which still fails the check because it is still in security/gnoi/systemz:

```
E0628 17:17:38.983546    2412 validate_paths.go:345] Feature paths inconsistent with YANG schema:
  file: /home/runner/work/featureprofiles/featureprofiles/feature/security/gnsi/authz/feature.textproto
  toplevel errors:
   gnsi_authz is inconsistent with path, want security_gnsi_authz
```

Reply:
done.